crates/ol/stf/src/tests_snark.rs

```diff
@@ -120,12 +283,28 @@ fn test_snark_account_deposit_and_withdrawal() {
     // Create the snark account update operation data
     let new_seqno = 1u64; // Next sequence number
     let new_state_root = Hash::from([2u8; 32]); // New state after update
-    let new_proof_state = ProofState::new(new_state_root, 0);
+
+    // Process the deposit message that was added during genesis
```

Comment:
see my comment above (line 234) -- are we processing the deposit now or did we process it earlier? what's going on here?
why this is here? we are in the process of some withdrawal, no?

we actually also not processing anything until the next comment, just declaring vars, is this supposed to by an umbrella `step n:` kind of comment that declares the following process (until a new `step n+1`)?